website/yarn.lock

```diff
@@ -11336,14 +11326,6 @@ universalify@^2.0.0:
   resolved "https://registry.yarnpkg.com/universalify/-/universalify-2.0.0.tgz#75a4984efedc4b08975c5aeb73f530d02df25717"
   integrity sha512-hAZsKq7Yy11Zu1DE0OzWjw7nnLZmJZYTDZZyEFHZdUhV8FkH5MCfoU1XMaxXovpyW5nq5scPqq0ZDP9Zyl04oQ==
 
-unleash-proxy-client@2.5.0:
```

Comment:
Is there a particular reason this gets removed?

Reply:
As far as I could find, it's not used anywhere in the (website/docs) codebase

Reply:
Yeah, this looks right. Just to give some context: we **did** use Unleash to control a feedback form previously, but that was .. probably years ago now, and we haven't used it since. I guess we (or rather: I) just forgot to remove the dependency.

Thanks for cleaning up after me!

Reply:
Thanks for the context @thomasheartman 🙏